### PR TITLE
Refreshed getting started 

### PR DESCRIPTION
**Pull request by sarah11918**
We've had some changes since this page was built, so here's a suggested draft to make it more in keeping with what's all going on now!  Open to suggestions... this is just to get the ball rolling.

### CODE REVIEW

**Review by aFuzzyBear — APPROVED** [review]
This looks good 😊

File: src/pages/en/getting-started.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ description: A basic intro to Astro.
 ---
 Static Site Generator  🚀  Bring your own Framework  🚀  Ship Less JavaScript
```
Comment (FredKSchott) [inline src/pages/en/getting-started.md:9]: No longer just a static site generator! Not sure what should go here instead, probably just a 1-2 sentence description of Astro?

**Review by FredKSchott — APPROVED** [review]
LGTM!

File: src/pages/en/getting-started.md (hunk 1 above)
Comment (FredKSchott) [reply src/pages/en/getting-started.md:9]: Working on this now for our own Repo readme, I'll copy something here when I have it. 

Also, Feel free to merge this without the change and I can create a PR for it later.

**Pull request merged**